server/services/premisesService.ts

```diff
@@ -40,7 +40,7 @@ export default class PremisesService {
 
     return premises
       .map(p => {
```

Comment:
Being pickie but `p` doesn't mean anything as a variable, maybe call it `singlePremises` or something?

Reply:
Yeah, pluralisation is hard 😊